it/test/controllers/manageAgent/AreYouSureMultipleISpec.scala

```diff
@@ -33,10 +33,15 @@ class AreYouSureMultipleISpec extends ISpecBase with HtmlComponentHelpers with L
 
   val titleText =
     "Are you sure you want Test Agent to act for you on the 2023 and 2017 rating lists? - Valuation Office Agency - GOV.UK"
+  val titleTextReval =
+    "Are you sure you want Test Agent to act for you on the 2026, 2023, and 2017 rating lists? - Valuation Office Agency - GOV.UK"
   val backLinkText = "Back"
   val captionText = "Manage agent"
   val headerText = "Are you sure you want Test Agent to act for you on the 2023 and 2017 rating lists?"
+  val headerTextReval = "Are you sure you want Test Agent to act for you on the 2026, 2023, and 2017 rating lists?"
   val forAllText = "For all your property valuations on the 2023 and 2017 rating lists, this agent will be able to:"
+  val forAllTextReval =
+    "For all your property valuations on the 2026, 2023, and 2017 rating lists, this agent will be able to:"
```

Comment:
Could you please add tests for the different combinations of 2 rating lists. I think I can see 2023 and 2017 is covered, but these aren't:
2026 & 2017
2026 & 2023